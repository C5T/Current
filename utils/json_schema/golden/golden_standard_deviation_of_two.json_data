{"x":1}
{"x":2}
{"x":3}
{"x":4}
{"x":5}
{"x":6}
{"x":7}
{"comment":"standard deviation must be two"}
