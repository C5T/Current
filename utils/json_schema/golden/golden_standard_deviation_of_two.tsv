Field	Type	Set	Unset/Null	Values	Details
Schema	Object	8	0	2 fields	x, comment
Schema.x	Integer	7	1	Mean 4, StdDev 2
Schema.comment	String	1	1	1 distinct value, "standard deviation must be two":1
